app/src/main/java/com/expidev/gcmapp/http/GmaApiClient.java

```diff
@@ -357,7 +364,7 @@ public JSONObject getDetailsForMeasurement(@NonNull final String measurementId,
         // build request
         final Request<Session> request = new Request<>(MEASUREMENTS + "/" + measurementId);
         request.params.add(param("ministry_id", ministryId));
-        request.params.add(param("mcc", mcc));
+        request.params.add(param("mcc", mcc.toLowerCase()));
```

Comment:
It definitely is making a difference.  I get all 0s back when using upper case, and data back when using lower case.